test/runtests.jl

```diff
@@ -2,8 +2,8 @@
 using Printf, SparseArrays, Test
 
 # our packages
-using LinearAlgebra, LinearOperators, NLPModels, QuadraticModels
 using QPSReader
+using LinearAlgebra, LinearOperators, NLPModels, QuadraticModels
```

Comment:
This is the section for our packages, in alphabetical order. LinearAlgebra should go above.